evml/keras/losses.py

```diff
@@ -1,16 +1,13 @@
 import tensorflow as tf
-from tensorflow.keras import backend as K
 import numpy as np
-from tensorflow.keras.utils import get_registered_name
 
 
 class DirichletEvidentialLoss:
```

Comment:
Should DirichletEvidentialLoss inherit from tensorflow.keras.losses.Loss like EvidentialRegressionLoss does?

Reply:
edited.